taxi/forms.py

```diff
@@ -0,0 +1,50 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.forms import ModelMultipleChoiceField
+
+from taxi.models import Driver
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data.get("license_number")
+
+        if len(license_number) != 8:
+            raise forms.ValidationError(
+                "License number must be at least 8 symbols"

```

Comment:
The validation message should state that the license number must be exactly 8 characters long, not "at least 8 symbols". This is a requirement from the task description.